<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BtcTransactionInfo</class>
 <widget class="QWidget" name="BtcTransactionInfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>721</width>
    <height>693</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Transaction details</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>25</y>
     <width>41</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>TxId:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>45</y>
     <width>62</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Amount:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>65</y>
     <width>41</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Fee:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>85</y>
     <width>101</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Confirmations:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>105</y>
     <width>271</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Conflicts (double spend, tx malleability):</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>135</y>
     <width>51</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Sent at:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>165</y>
     <width>81</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Received at:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>200</y>
     <width>62</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Details:</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableDetails">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>701</width>
     <height>192</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>Category</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Amount</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Destination</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Address</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Fee</string>
    </property>
   </column>
  </widget>
  <widget class="QComboBox" name="comboConflicts">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>100</y>
     <width>421</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelTxId">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>20</y>
     <width>491</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>(txid)</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelAmount">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>40</y>
     <width>181</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>0.0</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelFee">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>60</y>
     <width>241</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>0.0</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="numberConfirms">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>80</y>
     <width>64</width>
     <height>23</height>
    </rect>
   </property>
   <property name="segmentStyle">
    <enum>QLCDNumber::Filled</enum>
   </property>
  </widget>
  <widget class="QDateTimeEdit" name="dateTimeSent">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>130</y>
     <width>194</width>
     <height>27</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QDateTimeEdit" name="dateTimeReceived">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>160</y>
     <width>194</width>
     <height>27</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>430</y>
     <width>51</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Inputs</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableRawVout">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>570</y>
     <width>701</width>
     <height>101</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>vout</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>amount</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>to address</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>type</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>scriptPubKey</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>reqSigs</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>550</y>
     <width>51</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Outputs</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableRawVin">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>450</y>
     <width>701</width>
     <height>91</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>vout</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>txid</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>scriptSig</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>sequence</string>
    </property>
   </column>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
